x/validator/handler.go

```diff
@@ -76,10 +76,9 @@ func handleMsgSetTransactors(ctx sdk.Context, keeper Keeper, msg MsgSetTransacto
 
 	for _, transactor := range oldTransactors {
 		if _, exist := dedup[transactor.String()]; !exist {
-			keeper.RemoveAccount(ctx, transactor)
-			err := keeper.bankKeeper.SetCoins(ctx, candidate.ValAccount, sdk.NewCoins(sdk.NewCoin(common.QuotaCoinName, sdk.ZeroInt())))
+			err := keeper.RemoveTransactor(ctx, candidate.ValAccount, transactor)
```

Comment:
why need candidate.ValAccount here? Similarly, in line 69 above, shouldn't be `keeper.bankKeeper.SetCoins(ctx, transactor, ... `